appengine/images/guestbook/app.yaml

```diff
@@ -0,0 +1,17 @@
+# This file specifies your Python application's runtime configuration
```

Comment:
You can strip all of the comments out of this file. They're only needed for hello world.